geoportal/c2cgeoportal_geoportal/scripts/import_ngeo_apps.py

```diff
@@ -199,17 +134,12 @@ def main():
             </script>
     % endif
 """,  # noqa: E501
-                        data,
-                    )
-
-            data = re.sub("app", "{{package}}", data)
-            data = re.sub(re.escape("{{package}}lication"), "application", data)
-            data = re.sub(re.escape("{{package}}le"), "apple", data)
+                    data,
+                )
 
-# temporary disable ...
-#        if args.js:
-            # Full text search
-#            data = _sub(r"datasetTitle: 'Internal',", r"datasetTitle: '{{project}}',", data)
+        data = re.sub("app", "{{package}}", data)
+        data = re.sub(re.escape("{{package}}lication"), "application", data)
+        data = re.sub(re.escape("{{package}}le"), "apple", data)
```

Comment:
Except that, it looks always weird ?

Reply:
In one case we have `apple`, that's replaced by `{{package}}le`, this is to be back to apple ...